config/grafana/overview.json

```diff
@@ -1977,6 +1977,100 @@
         "align": false,
         "alignLevel": null
       }
+    },
+    {
+      "collapsed": false,
+      "datasource": null,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 110
+      },
+      "id": 84,
+      "panels": [],
+      "title": "Maintenance Windows",
+      "type": "row"
+    },
+    {
+      "datasource": null,
+      "description": "This indicates if the KLM was successful in reading the maintenance policy file from the mounted config map.",
```

Comment:
`"description": "Indicates whether the KLM successfully read the maintenance policy file from the mounted ConfigMap.",`